app/src/main/java/com/gatheringhallstudios/mhworlddatabase/features/decorations/detail/DecorationDetailFragment.kt

```diff
@@ -32,6 +31,31 @@ class DecorationDetailFragment : androidx.fragment.app.Fragment() {
     private val viewModel: DecorationDetailViewModel by lazy {
         ViewModelProviders.of(this).get(DecorationDetailViewModel::class.java)
     }
+    
+    override fun onCreate(savedInstanceState: Bundle?) {
+        setHasOptionsMenu(true)
+        super.onCreate(savedInstanceState)
+    }
+
+    override fun onCreateOptionsMenu(menu: Menu, inflater: MenuInflater) {
```

Comment:
This options menu creation and handling is repeated a bunch of times. Can this be moved into a superclass like `BookmarkableDetailFragment`?